### PR TITLE
Fix the values for option tokens (textDecoration, textCase, visibility)

### PR DESCRIPTION
**Pull request by honzatmn**
## Description
Change the textDecoration, textCase, and visibility token values to return correct CSS values for these CSS properties. 

## Checklist

- [x] I included doc comments for any utility I am adding
- [x] I wrote test for any utility I am adding and the tests cover use-cases reasonably
- [x] I made sure the helper was not built before
- [x] I run `npm run build` to make sure helper is buildable locally
- [x] I run `npm run test` to make sure all helpers still work
- [x] I run `npm run lint` to make sure there are no linting errors (run `npm run pretty` to autofix all issues)

### CODE REVIEW

File: src/transforms/CSSHelper.ts [hunk 1]
```diff
@@ -83,9 +83,11 @@ export class CSSHelper {
       case TokenType.string:
         return this.stringTokenValueToCSS((token as AnyStringToken).value, allTokens, options)
       case TokenType.textCase:
+        return this.optionTokenValueToCSS((token as AnyOptionToken).value, allTokens, options, token.tokenType)
       case TokenType.textDecoration:
+        return this.optionTokenValueToCSS((token as AnyOptionToken).value, allTokens, options, token.tokenType)
       case TokenType.visibility:
-        return this.optionTokenValueToCSS((token as AnyOptionToken).value, allTokens, options)
+        return this.optionTokenValueToCSS((token as AnyOptionToken).value, allTokens, options, token.tokenType)
```
Comment (tchief) [inline src/transforms/CSSHelper.ts:90]: All calls are same, why do we duplicate?
Just
```
case TokenType.textCase:
case TokenType.textDecoration:
case TokenType.visibility:
        return this.optionTokenValueToCSS((token as AnyOptionToken).value, allTokens, options, token.tokenType)
```

File: tests/transforms/CSSHelper.test.ts [hunk 2]
```diff
@@ -436,15 +443,30 @@ test('toCSS_optionToken_1', () => {
...
 test('toCSS_optionToken_2', () => {
   let option = {
     ...testOption,
     referencedTokenId: 'optionRef'
   }
-  expect(CSSHelper.optionTokenValueToCSS(option, tokens, testOptions)).toBe('var(--optionRef)')
+  expect(CSSHelper.optionTokenValueToCSS(option, tokens, testOptions, TokenType.textCase)).toBe('var(--optionRef)')
+})
+
+test('toCSS_optionToken_2', () => {
```
Comment (tchief) [inline tests/transforms/CSSHelper.test.ts:457]: Update test names

File: tests/transforms/CSSHelper.test.ts [hunk 3]
```diff
@@ -436,15 +443,30 @@ test('toCSS_optionToken_1', () => {
...
+  let option = {
+    ...testOptionTextDecoration
+  }
+  expect(CSSHelper.optionTokenValueToCSS(option, tokens, testOptions, TokenType.textDecoration)).toBe('underline')
+})
+
+test('toCSS_optionToken_2', () => {
```
Comment (tchief) [inline tests/transforms/CSSHelper.test.ts:464]: Update test names

Comment (honzatmn) [reply tests/transforms/CSSHelper.test.ts]: Good catch, thanks

File: tests/transforms/CSSHelper.test.ts (hunk 2 above)
Comment (honzatmn) [reply tests/transforms/CSSHelper.test.ts]: Good catch, thanks

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@supernovaio/export-helpers",
-  "version": "1.0.10",
+  "version": "1.0.11",
```
[the remark this answers is not in the document]
Comment (honzatmn) [reply package.json]: Updated

**Comment by honzatmn** [comment]
@tchief Can you please check after the changes? 🙏

**Review by tchief — APPROVED** [review]

**Pull request merged**